_fluo-1-2/administration/troubleshooting.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Troubleshooting
+category: administration
+order: 7
+---
+
+Steps for troubleshooting problems with Fluo applications.
+
+## Fluo application stops processing data
+
+1. Confirm that your application is running with the expected number of workers. 
+    ```bash
+    $ fluo list
+    Fluo instance (localhost/fluo) contains 1 application(s)
+
+    Application     Status     # Workers
+    -----------     ------     ---------
+    webindex        RUNNING        3
+    ```
+   Look for errors in the logs of any oracle or worker that has died.
+
+1. Run the `fluo wait` command to see if you application is processing notifications. 
+    ```bash
+    $ fluo wait -a webindex
+    [command.FluoWait] INFO : The wait command will exit when all notifications are processed
+    [command.FluoWait] INFO : 140 notifications are still outstanding.  Will try again in 10 seconds...
+    [command.FluoWait] INFO : 140 notifications are still outstanding.  Will try again in 10 seconds...
+    [command.FluoWait] INFO : 140 notifications are still outstanding.  Will try again in 10 seconds...
+    [command.FluoWait] INFO : 96 notifications are still outstanding.  Will try again in 10 seconds...
+    [command.FluoWait] INFO : 70 notifications are still outstanding.  Will try again in 10 seconds...
+    [command.FluoWait] INFO : 31 notifications are still outstanding.  Will try again in 10 seconds...
+    [command.FluoWait] INFO : All processing has finished!
+    ```
+   The number of notifications will increase as data is added to the application but they should eventually decrease
+   to zero and processing should finish.
```

Comment:
If notifications are not going down, user can scan for active notifications.  However the current way of scanning shows deleted notifications.  I opened apache/fluo#1026.  Even though the current way of scanning may not be ideal, its probably still worth mentioning along with a link to 1026.  I can add info about that in a follow on PR along with example output.

Reply:
Ok sounds good